01-Git_Bash_Make/main.cpp

```diff
@@ -0,0 +1,10 @@
+#include <iostream>
+
+using namespace std;
+
+int main(int argc, char** argv)
+{
+	cout << "Enter value: ";
```

Comment:
I think It is better to move the line output `cout << "Enter value: "` to the next commit, because functionality it is close to the 2nd commit.